tests/e2e/tidbcluster/serial.go

```diff
@@ -899,10 +899,12 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 				if err != nil {
 					return false, err
 				}
-				if secondTs-firstScaleTimestamp < 100 {
-					klog.Infof("tikv second scale's interval isn't meeting the interval requirement")
+				if secondTs == firstScaleTimestamp {
```

Comment:
Suggest adding log here so that we can know if TiKV does not scale before timeout.

Reply:
updated.